src/rebar3_hex_owner.erl

```diff
@@ -179,6 +178,12 @@ list(HexConfig, Package, State) ->
             ?PRV_ERROR({error, Package, Reason})
     end.
 
+owner_email(Owner) ->
+    case maps:get(<<"email">>, Owner, <<"">>) of
```

Comment:
I think the default here should be `undefined` otherwise you'll print out a blank new line in the end, thoughts?

Reply:
I guess we could change it. I just moved the code, so didn't think about prior reasoning behind it.